### ISSUE TITLE
Stories Pre-Publishing: too much white space under Publish Now button

### ISSUE THREAD

**Issue by startuptester** [wordpress-mobile/WordPress-iOS#16459]
### Expected behavior
In regular post pre-publishing, the Publish Now button is at the bottom 
![image](https://user-images.githubusercontent.com/66104034/117701739-d73f9580-b184-11eb-9b68-45ca81cdf5c6.png)

### Actual behavior
With a Story post, it looks like this
![IMG_3574](https://user-images.githubusercontent.com/66104034/117701823-ed4d5600-b184-11eb-9d20-f610c4d47f8a.PNG)

### Steps to reproduce the behavior
1. Create a Story Post 
2. Tap "Publish" 
3. Notice "Publish Now" button placement

originally reported by @kyleaparker

**Comment by rachelmcr** [comment]
I had a slightly different experience, but also related to the space under the Publish Now button, so I'll share it here:

In my case, the spacing looks great when I first see the bottom sheet. However, after I open one of the subsections (Visibility, Publish Date, etc.) and then go back, the keyboard is dismissed as the Publish button is overlapped by the iOS tab bar. (So there isn't _enough_ white space under the button.)

If I pull the bottom sheet up to avoid overlapping the tab bar, I can pull it all the way up so there's a ton of white space, which also seems odd.

Pre-publish bottom sheet|After selecting tags/category|After pulling the bottom sheet up
-|-|-
![IMG_6162](https://user-images.githubusercontent.com/8658164/118627387-94547200-b7c3-11eb-9ea9-a57687f841b4.PNG)|![IMG_6161](https://user-images.githubusercontent.com/8658164/118627400-961e3580-b7c3-11eb-9ec9-e592ee7aec83.PNG)|![IMG_6160](https://user-images.githubusercontent.com/8658164/118627423-9c141680-b7c3-11eb-80c7-7e0fcc648672.PNG)

Tested on iPhone 12 Mini, iOS 14.4.2, WPiOS 17.3

**Comment by dcalhoun** [comment]
I also experienced some oddities with the pre-publish bottom sheet. It is quite easy, through either dragging the sheet downward or visiting a sub-section, to end up with a bottom sheet that is too short to read any of its content. One can "recover" from this state by carefully dragging the bottom sheet and "flinging" it upwards, but it then results in a large amount of extra scrolling space. 

I did my best to capture my experience in the following video. 

Tested on an iPhone 11 Pro, iOS 14.6, WPiOS 17.6.0.3.

https://user-images.githubusercontent.com/438664/123323828-aed4e600-d4fb-11eb-9a04-2d6bdede1c7b.MP4
